velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

```diff
@@ -1270,6 +1270,14 @@ class ArrowBridgeArrayImportTest : public ArrowBridgeArrayExportTest {
 
     testArrowImport<int64_t, int128_t>(
         "d:5,2", {1, -1, 0, 12345, -12345, std::nullopt});
+    testArrowImport<int128_t, int128_t>(
```

Comment:
Does this test fail without this patch? I ran the test local without this patch, it passed.